README.md

```diff
@@ -1,13 +1,18 @@
 # Reden is What We Made
 
-[![Modrinth Downloads](https://img.shields.io/modrinth/dt/reden?style=flat-square&label=Modrinth)](https://modrinth.com/mod/reden)
-[![Discord](https://img.shields.io/discord/1140304794976792707?logo=discord&label=discord)](https://discord.gg/fCxmEyFgAd)
+[![Modrinth](https://img.shields.io/modrinth/dt/reden?style=flat-square&label=Modrinth)](https://modrinth.com/mod/reden)
+[![Discord Server](https://img.shields.io/discord/1140304794976792707?logo=discord&label=discord)](https://discord.gg/fCxmEyFgAd)
 [Official Website](https://redenmc.com)
 
 Redstone EDEN
 
 **English** | [简体中文](./README.zh-CN.md)
 
+## Before you start
```

Comment:
remove this, redenmc.com does not collect data from all user operations and we will send a privacy notice when the player launches reden the first time. on the other hand, i think this part should also contain the terms of service but it has not been complete yet